client/src/App.jsx

```diff
@@ -1,14 +1,14 @@
-import React from 'react';
-
-import { BrowserRouter } from 'react-router-dom'
-
-import { Auth } from './pages/Auth';
-
+import React, { useState } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { Public } from './routes/Public';
+import { Protected } from './routes/Protected';
```

Comment:
protected is a general name, can you give him more specifical name like 'ProtectedRoute' so we will able to create any other components with 'protected' name and be sure that its not auth